.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,78 @@
+name: Bug Report
+description: File a bug report to help improve qBittorrent user experience.
+body:
+  - type: markdown
+    attributes:
+     value: |
+      #### ADVISORY
+       "We do not support any versions older than the current release series" or 3rd party/forked versions e.g. `portableapps`/`Enhanced Edition`etc.
+       It is the responsibility of the user to **redact any sensitive information** in logs/config files.
+  - type: checkboxes
```

Comment:
Add an empty line above please.

Reply:
>I would prefer prefill the textarea to have consistent formatting.

Just a note on this, If it was prefilled - it would invalidate the true/false requirement.

We could be left with just:
      qBittorrent:
      Operating system:
      Qt:
      libtorrent-rasterbar:

I added a comment in description about how we would like it formatted & added them as a placeholder instead.